src/sentry/static/sentry/app/views/settings/components/forms/form.jsx

```diff
@@ -109,7 +109,8 @@ export default class Form extends React.Component {
         this.model.getData(),
         this.onSubmitSuccess,
         this.onSubmitError,
-        e
+        e,
+        this.model.setFormSaving.bind(this.model)
```

Comment:
the order of these args is not ideal, but didn't want to break backwards compatibility :(